### PR TITLE
Fix multiple duplicate person records created in openmrs 

### LINKED ISSUE #5: Update the patient create to skip Identifier Types with null values
Currently an identifier with a null value if created in the patient payload. This thus leads to the patient not being created and everytime the server tries to recreate the patient it fails leading to multiple person created with the same details

### PR DESCRIPTION
**Pull request by ndegwamartin**
Fixes [Issue 5](https://github.com/OpenSRP/opensrp-server-connector/issues/5)

### CODE REVIEW

File: src/main/java/org/opensrp/connector/openmrs/schedule/OpenmrsSyncerListener.java [hunk 1]
```diff
@@ -48,16 +43,11 @@
...
-	@Autowired
-	private Dhis2TrackCaptureConnector dhis2TrackCaptureConnector;
-	
 	@Autowired
 	public OpenmrsSyncerListener(ScheduleService opensrpScheduleService, ActionService actionService, ConfigService config,
 	    ErrorTraceService errorTraceService, PatientService patientService, EncounterService encounterService,
 	    ClientService clientService, EventService eventService) {
 		
-		this.opensrpScheduleService = opensrpScheduleService;
-		this.actionService = actionService;
```
Comment (dubdabasoduba) [inline src/main/java/org/opensrp/connector/openmrs/schedule/OpenmrsSyncerListener.java:60]: Why did you remove this and still left `opensrpScheduleService` & `actionService` as method params?

File: src/main/java/org/opensrp/connector/openmrs/schedule/OpenmrsSyncerListener.java [hunk 2]
```diff
@@ -48,16 +43,11 @@
 	
 	private final ClientService clientService;
 	
-	@Autowired
-	private Dhis2TrackCaptureConnector dhis2TrackCaptureConnector;
```
Comment (dubdabasoduba) [inline src/main/java/org/opensrp/connector/openmrs/schedule/OpenmrsSyncerListener.java:52]: This seems to be DHIS specific while remove it?

**Review by dubdabasoduba — CHANGES_REQUESTED** [review]

File: src/main/java/org/opensrp/connector/openmrs/schedule/OpenmrsSyncerListener.java (hunk 1 above)
Comment (ndegwamartin) [reply src/main/java/org/opensrp/connector/openmrs/schedule/OpenmrsSyncerListener.java:60]: These are **private** fields which are unused anywhere in this class hence they are not required

File: src/main/java/org/opensrp/connector/openmrs/schedule/OpenmrsSyncerListener.java (hunk 2 above)
Comment (ndegwamartin) [reply src/main/java/org/opensrp/connector/openmrs/schedule/OpenmrsSyncerListener.java:52]: This field also has a **private** access privileges but is unused anywhere in this class hence it is not required either

**Review by dubdabasoduba — APPROVED** [review]

**Pull request merged**